_data/anniezou/anniezou.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Annie Zou # No longer than 18 characters
+institution: Princeton University 🚩 # no longer than 58 characters
+profile_pic: headshot.png # Name and extension of your profile picture(ex. mona.png)
+quote: "there's magic that brought us together and that magic is called hope" --detective pikachu # no longer than 100 characters
```

Comment:
```suggestion
quote: "there's magic that brought us together and that magic is called hope" # no longer than 100 characters
```
The text after the quote is breaking the markdown file. You can either remove the  `--detective pikachu` or the `"`